lib/web_ui/lib/src/ui/geometry.dart

```diff
@@ -1552,6 +1552,24 @@ class RRect {
     return true;
   }
 
+  /// Checks if other RRect is fully contained within this.
+  bool fullyContains(RRect other) {
```

Comment:
I would not make this a public API on RRect unless you also intend to implement it for the other dart:ui. Since you're only implementing it for a single method I would keep it inline there

Reply:
+1